package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "fonts",
   "main": "./lib/fonts",
-  "version": "1.2.0",
+  "version": "1.3.0",
```

Comment:
You don't need to update this by hand, `apm` manages the version numbers upon release.